mobile/contacts.go

```diff
@@ -1,13 +1,21 @@
 package mobile
 
 import (
-	"github.com/pkg/errors"
+	"encoding/json"
+	"errors"
+
 	"github.com/textileio/textile-go/core"
+	"github.com/textileio/textile-go/repo"
 )
 
 // AddContact calls core AddContact
-func (m *Mobile) AddContact(id string, address string, username string) error {
-	return m.node.AddContact(id, address, username)
+func (m *Mobile) AddContact(contact string) error {
+	var model *repo.Contact
```

Comment:
Mobile change: `AddContact` takes a JSON string that is the entire contact object... this would be this thing that gets sent to the peer exchange @andrewxhill